src/xmldiffng/fmes2.py

```diff
@@ -0,0 +1,304 @@
+from xmldiff import fmes
+from xmldiff.fmes import FmesCorrector
+
+class FmesCorrectorRNG(FmesCorrector):
+    def __init__(self, formatter, rngdict=None, f=0.6, t=0.5): # f=0,59
+        rngjson = '/suse/fweiss/Documents/werkstudent/bachelorarbeit/xmldiffng/contrib/rng.json'
```

Comment:
That path shouldn't be hardcoded. Better provide it via a CLI interface. Change the signature of the constructor to:

```python
def __init__(self, formatter, rngjson, rngdict=None, f=0.6, t=0.5):
```

The `rngjson` is then passed when you instantiate your class. **IF** you really have to, use a the methods from the `os.path` module and `__file__`.